### PR TITLE
Adding motivation for tutorial

### LINKED ISSUE #16: Why another Python tutorial?
I'm creating this issue to be the discussion place on the motivation behind this tutorial.

This is what I've written so far:

Why *this* Python tutorial?

What makes this Python tutorial unique is that is has been designed specifically to meet the needs of, and feedback from, atmospheric and oceanic scientists making the transition with the NCAR-wide [pivot-to-Python](https://www.ncl.ucar.edu/Document/Pivot_to_Python/). By spending the first day on pure Python without importing any additional packages, our beginner tutorial addresses the concerns that most tutorials pick up speed too quickly. By using coding examples with real atmospheric datasets and questions, the skills and techniques taught are easily applied to actual workflows. Intermediate and advanced tutorials were developed by working closely with scientists to transform their old code into Python using the most up-to-date tools for advanced computing. We hope that this tailored approach to teaching and sharing computational tools effectively addresses the concerns and needs of the geoscience community.

### PR DESCRIPTION
**Pull request by jukent**
Closes #16 
Screen Shot 2020-06-17 at 12 38 00 PM

### CODE REVIEW

File: source/_templates/index.html [hunk 1]
```diff
@@ -6,7 +6,26 @@
         <div class="jumbotron">
           <div class="container">
             <h1>NCAR Python Tutorial</h1>
-            <p>Let's learn</p>
+            <p>
+                What makes this Python tutorial unique is that is has been designed specifically
```
Comment (kmpaul) [inline source/_templates/index.html:10]:
```suggestion
                What makes this Python tutorial unique is that it has been designed specifically 
```

File: source/_templates/index.html [hunk 2]
```diff
@@ -6,7 +6,26 @@
         <div class="jumbotron">
           <div class="container">
             <h1>NCAR Python Tutorial</h1>
-            <p>Let's learn</p>
+            <p>
+                What makes this Python tutorial unique is that is has been designed specifically 
+                to meet the needs of, and feedback from, atmospheric and oceanic scientists 
+                making the transition with the NCAR-wide 
+                <a class="reference external" href = "https://www.ncl.ucar.edu/Document/Pivot_to_Python/"> pivot-to-Python</a>.
```
Comment (kmpaul) [inline source/_templates/index.html:13]:
```suggestion
                <a class="reference external" href = "https://www.ncl.ucar.edu/Document/Pivot_to_Python/">pivot-to-Python</a>. 
```

File: source/_templates/index.html [hunk 3]
```diff
@@ -6,7 +6,26 @@
         <div class="jumbotron">
           <div class="container">
             <h1>NCAR Python Tutorial</h1>
-            <p>Let's learn</p>
+            <p>
+                What makes this Python tutorial unique is that is has been designed specifically 
+                to meet the needs of, and feedback from, atmospheric and oceanic scientists 
+                making the transition with the NCAR-wide 
+                <a class="reference external" href = "https://www.ncl.ucar.edu/Document/Pivot_to_Python/"> pivot-to-Python</a>. 
+                In particular, this tutorial should be useful for any scientist who already knows how to program
```
Comment (kmpaul) [inline source/_templates/index.html:14]:
```suggestion
                In particular, this tutorial should be useful to any scientist who already knows how to program 
```

File: source/_templates/index.html [hunk 4]
```diff
@@ -6,7 +6,26 @@
...
             <h1>NCAR Python Tutorial</h1>
-            <p>Let's learn</p>
+            <p>
+                What makes this Python tutorial unique is that is has been designed specifically 
+                to meet the needs of, and feedback from, atmospheric and oceanic scientists 
+                making the transition with the NCAR-wide 
+                <a class="reference external" href = "https://www.ncl.ucar.edu/Document/Pivot_to_Python/"> pivot-to-Python</a>. 
+                In particular, this tutorial should be useful for any scientist who already knows how to program 
+                in some other language but is taking up Python for the first time. By spending the 
+                first day on pure Python without importing any additional packages, our beginner
```
Comment (kmpaul) [inline source/_templates/index.html:16]: > first day

Do we want to say this?  I think this tutorial should be described as self-paced, which I think means we should drop suggestions that the material is timed in any way.  There is probably other content like this throughout the tutorial.

Comment (jukent) [reply source/_templates/index.html:16]: I will change it to first course.

Comment (jukent) [reply source/_templates/index.html]: "Part 1 is designed to take one work day, but you may move through the content
much slower or more quickly."  This is the only other reference to time.

**Comment by kmpaul** [comment]
I'm also wondering, now, whether this is a lot of text for the landing page.  Maybe most of this *big* paragraph should go at the top of the beginner tutorial.  And then a brief summary (just a few lines) should go on the landing page?

File: source/_templates/index.html (hunk 4 above)
Comment (kmpaul) [reply source/_templates/index.html]: ...Ok.  That can be fixed easily, then.

This also suggests, though, that we have text throughout that refers to "Parts," which doesn't really fit the actual structure any more.  It's really just 1 big part, right?

Comment (kmpaul) [reply source/_templates/index.html]: ...Or, more specifically, there are no numbered parts, anymore.

**Comment by jukent** [comment]
> I'm also wondering, now, whether this is a lot of text for the landing page. Maybe most of this _big_ paragraph should go at the top of the beginner tutorial. And then a brief summary (just a few lines) should go on the landing page?

I was having a similar thought. I'll move it.

**Comment by jukent** [comment]
Screen Shot 2020-06-17 at 12 38 00 PM

File: source/_templates/index.html (hunk 4 above)
Comment (jukent) [reply source/_templates/index.html]: I just removed all references to numbered parts of the tutorial.

File: source/_templates/index.html [hunk 5]
```diff
@@ -6,26 +6,7 @@
...
-                that anyone with programming experience already knows. This tutorial attempts to hit 
-                the sweet spot between too high-level and too low-level. By using coding examples 
-                with real atmospheric datasets and questions, the skills and techniques taught are 
-                easily applied to actual atmospheric or oceanic workflows. The intermediate and 
-                advanced tutorials were developed by working closely with scientists to transform their 
-                old code into Python using the most up-to-date tools for advanced computing. 
-                We hope that this tailored approach to teaching and sharing computational tools 
-                effectively addresses the concerns and needs of the geoscience community.
-              </p>
+            <p> Let's learn </p>
```
[the remark this answers is not in the document]
Comment (jukent) [reply source/_templates/index.html]: I think I like the clean look of not much text at all on the home page.

**Comment by kmpaul** [comment]
Working in master branch. :smile:

**Pull request closed without merging**